web/docs/understand/operators/sources/export.md

```diff
@@ -0,0 +1,37 @@
+# export
+
+Retrieves events from a VAST node. The dual to [`import`](../sinks/import.md).
+
+## Synopsis
+
+```
+export
+```
+
+## Description
+
+The `export` operator retrieves events from a VAST node.
+
+:::note Flush to disk
+Pipelines starting with the `export` operator do not access events that are not
+written to disk.
+
+We plan to change this behavior in the near future. Until then, we recommend
+running `vast flush` before exporting events to make sure they're available for
```

Comment:
```suggestion
running `tenzir-ctl flush` before exporting events to make sure they're available for
```

Reply:
I replaced the command in https://github.com/tenzir/vast/pull/3193/commits/45aaa7cf4310261cd85e18870d054ffae86b2149